cmd/cmd.go

```diff
@@ -0,0 +1,90 @@
+package cmd
```

Comment:
I think the mitchellh/cli package or go flag package implements a lot of this functionality?